user-licence-lifecycle-manager/src/test/java/de/intension/rest/LicenceResourceIT.java

```diff
@@ -124,11 +130,30 @@ private Pair<String, String> getDatabaseEntry() throws Exception {
 
     @Test
     void should_return_404_when_hmac_id_does_not_exist() throws Exception {
-        var request = HttpRequest.newBuilder(URI.create("http://localhost:" + keycloak.getHttpPort() + "/auth/realms/fwu/licences/invalid-hmac-id")).GET().build();
+        var accessToken = getAccessToken();
+        var request = HttpRequest.newBuilder(URI.create("http://localhost:" + keycloak.getHttpPort() + "/auth/realms/fwu/licences/invalid-hmac-id")).GET().header("Authorization", "Bearer " + accessToken).build();
         var response = httpClient.send(request, HttpResponse.BodyHandlers.discarding());
         assertEquals(404, response.statusCode());
     }
 
+    private String getAccessToken() throws Exception {
+        var parameters = GRANT_TYPE + "=" + PASSWORD + "&" + USERNAME + "=misty&" + PASSWORD + "=test&" + CLIENT_ID + "=admin-cli";
+        var request = HttpRequest.newBuilder(URI.create(keycloak.getAuthServerUrl() + "/realms/fwu/protocol/openid-connect/token"))
+                .POST(HttpRequest.BodyPublishers.ofString(parameters))
+                .header("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
nitpick: use constants - `HttpHeaders.CONTENT_TYPE` and `MediaType.APPLICATION_X_WWW_SOMETHING`